core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -4539,13 +4541,13 @@ public void run() {
                     if (isQuietingDown()) {
                         servletContext.setAttribute("app", new HudsonIsRestarting());
                         // give some time for the browser to load the "reloading" page
-                        LOGGER.info("Restart in 10 seconds");
+                        lifecycle.onStatusUpdate("Restart in 10 seconds");
```

Comment:
This gives us fancy status notifications in `systemctl status`:

```
$ sudo systemctl status jenkins
● jenkins.service - Jenkins Continuous Integration Server
     Loaded: loaded (/lib/systemd/system/jenkins.service; enabled; vendor preset: enabled)
    Drop-In: /etc/systemd/system/jenkins.service.d
             └─override.conf
     Active: active (running) since Sat 2022-01-29 16:52:16 PST; 1min 4s ago
   Main PID: 169136 (java)
     Status: "Restart in 10 seconds"
      Tasks: 59 (limit: 38237)
     Memory: 2.9G
     CGroup: /system.slice/jenkins.service
             └─169136 /usr/bin/java -Djava.awt.headless=true -jar /usr/share/java/jenkins.war --webroot=/var/cache/jenkins/war --httpPort=8080
```